app/src/main/java/com/monkeyteam/chimpagne/ui/navigation/NavigationActions.kt

```diff
@@ -12,6 +12,8 @@ object Route {
   const val FIND_AN_EVENT_SCREEN = "FindAnEvent"
   const val LOADING = "loading"
   const val EVENT_CREATION_SCREEN = "EVENT_CREATION_SCREEN"
+  const val MY_EVENTS_SCREEN = "myEvents"
+  const val VIEW_DETAIL_EVENT_SCREEN = "viewDetailEVentScreen"

```

Comment:
typo: viewDetailEVentScreen -> viewDetailEventScreen
It seems PascalCase has been mostly adopted for our routes

Reply:
fixed as well!